lib/views/base/base_screen.dart

```diff
@@ -51,7 +63,15 @@ abstract class BaseScreen<T extends GetxController> extends GetView<T> {
 
   // SafeArea의 색상을 정의하는 메서드
   @protected
-  Color? get unSafeAreaColor => Colors.black;
+  Color? get unSafeAreaColor => Colors.transparent;
+
+  // SafeArea의 색상을 정의하는 메서드
```

Comment:
```suggestion
  /// SafeArea의 색상을 정의하는 메서드
```
주석을 /// 로 하면 나중에 마우스 올렸을 때, 어떤 속성인지 한번에 보기 쉬워용